packages/link/lib/TextLink.jsx

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
-import Link, {linkPropTypes} from './Link';
+import Link from './Link';
 import * as styles from './text-link.module.scss';
 
+/**
+ * A version of the Link component that styles a hyperlink composed of
+ * an icon and text. It can also be icon-only or text-only.
+ *
+ * Any props not defined by this component are passed through to the Link component.
+ */
```

Comment:
Do you think it's worth it to add linting rules that require documentation, either for components or props?

Reply:
that's a really good question. i think requiring documentation for the component (this level where your comment is) is a no-brainer, but i don't know if we should require documentation for all props. here, i didn't document the `children` or `id` props because i considered them self-explanatory, but maybe that's an oversight and i just assumed they don't need documentation